Firestore/core/src/remote/grpc_streaming_reader.h

```diff
@@ -99,8 +103,11 @@ class GrpcStreamingReader : public GrpcCall, public GrpcStreamObserver {
   std::unique_ptr<GrpcStream> stream_;
   grpc::ByteBuffer request_;
 
-  Callback callback_;
-  ResponsesT responses_;
+  size_t expected_response_count_;
+  bool callback_fired_ = false;
+  MessagesCallback docs_callback_;
```

Comment:
Nit: Can we make this (and all references to message) "ResponsesCallback" (and rename all references to message). We are mixing the terms messages and responses freely, when I think they are all the same. The instance variable also still needs to renaming (from docs_callback_).

Reply:
My apologies, I should change it together with the function type